snap/snapcraft.yaml

```diff
@@ -69,11 +66,22 @@ parts:
     override-build: |
       ${SNAP}/libexec/snapcraft/craftctl default
 
-      version="$(python3 setup.py --version)"
+      # craft-parts changed the python symlink in venv, and unusable at the time
```

Comment:
It feels like this override-build script is worse than just having an "empty" `setup.py` file. If we kept the setup.py, we wouldn't have to hack around with symlinks here nor build the project manually.